src/index.html

```diff
@@ -1,14 +1,42 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <a href="#home">
+          <img
+            class="image"
+            src="./images/logo.png"
+            alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <a class="link is-active" href="#">Apple</a>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#">Apple</a>
```
Нащадком ul може бути li, а не `<a>`. Такий код не валідний